integration-tests/tests/errors/errors.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Errors test</title>
+
+	<%- renderAgent({debug: true}) %>
+</head>
+<body>
+	<h1>FETCH</h1>
+	<script>
+		fetch('/some-data').then(response => response.json()).then(console.log);
```

Comment:
Was this just a copy+paste that isn't used any more?

Reply:
Yes 😕